### PR TITLE
Adding new BidAdapter 'eywamedia'

### PR DESCRIPTION
**Pull request by Sharatheywa**
## Type of change

- [x] New bidder adapter   

## Description of change

- test parameters for validating bids
```
{
  bidder: 'eywamedia',
  params: {
    publisherId: "1234_abcd"
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
sharath@eywamedia.com
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/pull/1418

### CODE REVIEW

File: test/spec/modules/eywamediaBidAdapter_spec.js [hunk 1]
```diff
@@ -0,0 +1,257 @@
+import { expect } from 'chai';
+import { spec } from 'modules/eywamediaBidAdapter';
+import { newBidder } from 'src/adapters/bidderFactory';
+import * as utils from 'src/utils';
```
[the remark this answers is not in the document]
Comment (Sharatheywa) [reply test/spec/modules/eywamediaBidAdapter_spec.js]: Actually not required, it was for testing.

**Comment by Fawke** [comment]
Hey @Sharatheywa

I was in the process of testing the bid response, but, for some reason, the call to your Ad server is failing, it's behaving very erratically. I've attached a screenshot.

Screenshot 2019-08-05 at 8 01 37 PM

The first request to your ad server, with HTTP method **OPTIONS** is returning 200, but the actual post call is just stalling out.

Sometimes when I refresh, it starts working again. I am testing with the test parameter you gave in the PR description.

**Comment by Sharatheywa** [comment]
Hi @Fawke,
May it was due to little delay in the response time, I've optimised it. Can you please check again.

**Comment by Fawke** [comment]
@Sharatheywa Thanks for making the change, the ad seems to be loading consistently now. One more thing, I checked your response headers coming back from the call to your ad server and notice that the `content-encoding` is not set to `gzip`. Can you GZIP your response?

**Comment by Sharatheywa** [comment]
> @Sharatheywa Thanks for making the change, the ad seems to be loading consistently now. One more thing, I checked your response headers coming back from the call to your ad server and notice that the `content-encoding` is not set to `gzip`. Can you GZIP your response?

Done.

**Comment by Sharatheywa** [comment]
Please check now

**Pull request merged**